CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changes to the Mapbox Navigation SDK for iOS
 
+## main
+
+### Location Tracking
+
+* Added `customActivityType` to `MapboxNavigationService` initialization to allow overriding default activity type for location updates during this navigation session. Changed default activity type from `automotiveNavigation` to `otherNavigation` for `.automobile` and `.automobileAvoidingTraffic` profiles.  ([#4068](https://github.com/mapbox/mapbox-navigation-ios/pull/4068))
+* Added `NavigationSettings.navigatorPredictionInterval` to control how far ahead Navigator will predict user current position. ([#4072](https://github.com/mapbox/mapbox-navigation-ios/pull/4072))
+
```

Comment:
We should move the content of the `## main` to `## 2.7.0`, if we want to include them to the 2.7 final release.

Reply:
thanks for catching that. fixed!